code/style.css

```diff
@@ -2,6 +2,327 @@
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    margin-top: 1rem;
+    text-decoration: underline;
+    font-weight: bold;
+    color: orange;
+}
+
+.container input{
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
+      
+}
+
+/* the checkmark in the raido button and hover */
+.checkmark{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 25px;
+  width: 25px;
+  background-color: #eee;
+  border-radius: 50%;
+}
+
+
+.container:hover input ~ .checkmark {
+    background-color: #ccc;
+  }
+  
+  
+  .container input:checked ~ .checkmark {
+    background-color: orange;
+  }
+  
+ 
+  .checkmark:after {
+    content: "";
+    position: absolute;
+    display: none;
+  }
+  
+  
+  .container input:checked ~ .checkmark:after {
+    display: block;
+  }
+  
+  
+  .container .checkmark:after {
+    left: 9px;
+    top: 5px;
+    width: 5px;
+    height: 10px;
+    border: solid black;
+    border-width: 0 3px 3px 0;
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+  }
+
+        /* The styling of the checkmark */
+        .terms{
+          display:block;
+          padding: 1rem;
+          border: 1px solid black;
+          background:white;
+          max-width: 100%;
+          margin-top:1rem;
+        }
+  
+        .termscheck{
+          height: 1rem;
+          width: 1rem;
+          vertical-align: middle;
+          
+        }
+
+ 
+
+       /* FAQ Section */
+       .accordion{
+           background: black;
+           display: flex;
+           flex-direction: column;
+           align-items: center;
+           
+          }
+
+/* FAQ div and then the faq h1*/
+      .faq{
+        display:flex;
+        width: 100%;
+        height: 20%;
+        background:black;
+        justify-content: center;
+        background-size:cover;
+        
+}
+
+.faqtext{
+  text-decoration: underline;
+  color: white;
+  font-weight: bold;
+}
+
+/* the FAQ buttons with hover */
+       .accordion-button {
+        background: orange;
+        color: black;
+        cursor: pointer;
+        padding: 18px;
+        width: 100%;
+        outline: none;
+        transition: 0.4s;
+        text-align: center;
+        margin-top: 5px;
+         
+    }
+
+    .accordion-button:after {
+        content: '\002B';
+        color: black;
+        font-weight: bold;
+        float: right;
+        margin-left: 5px;
+        
+      }
+      
+      .active:after {
+        content: "\2212";
+      }
+      
+      
+      .accordion-button:hover{
+          background: white;
+        
+      }
+      
+      /* the FAQ answers */
+      .panel {
+        padding: 0 18px;
+        background: white;
+        max-height: 5rem;
+        display: none;
+        overflow: hidden;
+        transition:max-height 0.2s ease-out;
+      }
+
+
+
+
+/* should've added more @media but I did not have time to do that :( */
```

Comment:
aaah, I see, even though you didn't have more media queries it works quite well overall!